app/models/movie.rb

```diff
@@ -1,31 +1,27 @@
 class Movie < ApplicationRecord
 
-    belongs_to :user
-    has_many :review, dependent: :delete_all
-    has_many :categorizations, dependent: :delete_all
-    has_many :genres, through: :categorizations
+  belongs_to :user
+  has_many :review, dependent: :delete_all
+  has_many :categorizations, dependent: :delete_all
+  has_many :genres, through: :categorizations
 
-    validates :title, presence:true
-    validates :overview, presence:true
-    validates :poster_path, presence:true
-    validates :release_date, presence:true
+  validates :title, presence:true
+  validates :overview, presence:true
+  validates :poster_path, presence:true
+  validates :release_date, presence:true
 
-    accepts_nested_attributes_for :categorizations, :reject_if => :all_blank, :allow_destroy => true
-    accepts_nested_attributes_for :genres
-    # def facebook
-    #  @movies = Facebook.get_object(current_user.token, '/me/movies?fields=name')
-    #end
-    #
-    #
-   def self.search(pattern)
+  accepts_nested_attributes_for :categorizations, :reject_if => :all_blank, :allow_destroy => true
+  accepts_nested_attributes_for :genres
+
+  def self.search(pattern)
     if pattern.blank?  # blank? covers both nil and empty string
       all
     else
-      where('title LIKE ?', "%#{pattern}%")
+      where('lower(title) LIKE lower(?)', "%#{pattern}%")
```

Comment:
@sdsunjay this can be simplified

`where(arel_table[:title].lower.matches(pattern))`

Reply:
does arel work by default? No extra config or gems necessary?

Reply:
@sdsunjay yes

Reply:
Okay, I will make the change and use what you have suggested